cacher/cacher_test.go

```diff
@@ -103,8 +103,11 @@ var _ = Describe("Cacher", func() {
 			Eventually(request.Responder).Should(Receive(&receivedResp))
 
 			queryTxResp := receivedResp.Result.(jsonrpc.ResponseQueryTx)
-			Expect(queryTxResp.Tx).To(Equal(queryTx.Tx))
+			Expect(queryTxResp.Tx.Hash).To(Equal(queryTx.Tx.Hash))
+			Expect(queryTxResp.Tx.Input).To(Equal(queryTx.Tx.Input))
+			Expect(queryTxResp.Tx.Output).NotTo(Equal(queryTx.Tx.Output))
 			res, err := queryTxResp.Tx.Output.MarshalJSON()
```

Comment:
```suggestion
			res, err := queryTxResp.Tx.Output.MarshalJSON()
			Expect(err).ToNot(HaveOccurred())
```

Reply:
Added this check directly to `release/0.4.8`.